### PR TITLE
Add traffic_analysis.csv

### PR DESCRIPTION
**Pull request by as2388**
Uses RQA timestamps to produce estimates for the number of people triggered to engage by each show (or advert/promo etc.), filtered by opt-ins and relevants. This initially uses the broadcast schedule for weeks 1-3, but we will extend with the remaining weeks once the broadcast schedule is known.

To configure, we just need to copy the broadcast schedule into the new, optional AutomatedAnalysis.TrafficLabels section of the configuration json. If the traffic analysis code in automated_analysis.py looks good, I'll adapt it and move it over to CoreData. For now this is quite low level for maximum flexibility, requiring explicit setting of each start/end date each week.

Sample output: https://drive.google.com/file/d/1OFQODtmt0A4fU-2mQ56C5drORRgwnzlk/view?usp=sharing.

### CODE REVIEW

File: configuration/sms_pipeline_config.json [hunk 1]
```diff
@@ -182,7 +182,28 @@
...
+      {"StartDate": "2020-11-12T14:30+03:00", "EndDate": "2020-11-12T15:30+03:00", "Label": "E02 Dalsan/Danan"},
+      {"StartDate": "2020-11-12T15:00+03:00", "EndDate": "2020-11-12T16:00+03:00", "Label": "E02 Risala"},
+      {"StartDate": "2020-11-12T11:30+03:00", "EndDate": "2020-11-12T12:30+03:00", "Label": "E02 Star"},
+      {"StartDate": "2020-11-12T16:30+03:00", "EndDate": "2020-11-12T17:30+03:00", "Label": "E02 Mustaqbal"},
+
+      {"StartDate": "2020-11-15T16:30+03:00", "EndDate": "2020-11-15T17:30+03:00", "Label": "E03 Ad"},
+      {"StartDate": "2020-11-18T16:30+03:00", "EndDate": "2020-11-18T17:30+03:00", "Label": "E03 Imaqal Ad"},
+      {"StartDate": "2020-11-19T14:30+03:00", "EndDate": "2020-11-19T15:30+03:00", "Label": "E03 Dalsan/Danan"},
+      {"StartDate": "2020-11-19T15:00+03:00", "EndDate": "2020-11-19T16:00+03:00", "Label": "E03 Risala"},
+      {"StartDate": "2020-11-19T11:30+03:00", "EndDate": "2020-11-19T12:30+03:00", "Label": "E03 Star"},
```
Comment (IsaackMwenda) [inline configuration/sms_pipeline_config.json:204]: if there is no other radio show right after we should include at least 30 min buffer after the show. In ke have observed a substantial number of messages streaming in 30 min -1 hour after the radio show.

File: automated_analysis.py [hunk 2]
```diff
@@ -226,6 +227,33 @@
                 "Participants": participants
             })
 
+    # Export traffic analysis
+    if pipeline_configuration.automated_analysis.traffic_labels is not None:
+        traffic_analysis = []
+        for tl in pipeline_configuration.automated_analysis.traffic_labels:
```
Comment (IsaackMwenda) [inline automated_analysis.py]:
```suggestion
        for traffic_label in pipeline_configuration.automated_analysis.traffic_labels:
```
easier to read.

**Review by IsaackMwenda — APPROVED** [review]

File: configuration/sms_pipeline_config.json (hunk 1 above)
Comment (as2388) [reply configuration/sms_pipeline_config.json:204]: Thanks - this actually does already include a 30 minute buffer, as IBTCI's shows are only 30 minutes long.

File: automated_analysis.py (hunk 2 above)
Comment (as2388) [reply automated_analysis.py]: Changed

**Pull request merged**